src/LAMMPS.jl

```diff
@@ -233,6 +260,89 @@ function command(lmp::LMP, cmd::Union{String, Array{String}})
     check(lmp)
 end
 
+"""
+    create_atoms(
+        lmp::LMP, x::Matrix{R1}, id::Vector{I1}, types::Vector{I2};
+        v::Union{Nothing,Matrix{R2}}=nothing,
+        image::Union{Nothing,Vector{I3}}=nothing,
+        bexpand::Bool=false
+    ) where {R1 <: Real, R2 <: Real, I1 <: Integer, I2 <: Integer, I3 <: Integer}
+
+Create atoms for a LAMMPS instance. x contains the atom positions and should be a 3 by n
+Matrix{Float64}, where n is the number of atoms. v contains the associated velocities and 
+should be the same size and type. id, types, and image should all be Vector{Int32} with a 
+length of n.
+"""
```

Comment:
A more detailed explaination of the parameters would be nice. Specifically on what `dexpand` does, the format of the image flags, and an explaination that types specifies the individual atom types.

Reply:
I added some more info in the docstring. Let me know what you think and I can add or change anything you'd like.